smack-core/src/test/java/org/jivesoftware/smack/test/util/CharSequenceEquals.java

```diff
@@ -1,47 +0,0 @@
-/**
```

Comment:
Why did you delete this class?

Reply:
Yep, why did this class go? If it's not longer used then we should consider removing it, but with an extra commit.